src/styles/block/catalog--card-stule.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  height: 408px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 0 36px;
+    text-align: center;
+  }
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use zero before the decimal point for values between 0 and 1. For example, `margin: 0.5em;` instead of `margin: .5em;`. Please check if there are any such cases in your styles and correct them.